template (interactive object): choice

JaskRendix opened the pull request:
connected to #1772 

same method, it makes disappear the tuxeball once chosen. 

![Screenshot from 2023-04-22 19-03-28](https://user-images.githubusercontent.com/64643719/233797825-a162cade-1f9f-46d4-9d7d-b839dc02bb09.png)
![Screenshot from 2023-04-22 19-01-54](https://user-images.githubusercontent.com/64643719/233797828-880ddc18-02d8-4814-88c1-6d2ad26b9fc6.png)

work in progress, black, isort, no new typehints

JaskRendix committed:
choice

JaskRendix commented:
@Qiangong2 with the last PR we can make disappear the object (at the moment only a few), I have already done some tests and as you can see from the screenshots above, both tuxeballs are going to disappear if chosen (one player, one professor).

Are you interested? If yes, do you have a specific set of colors?
Alternatively you can put three identical tuxeballs without distinction (all red, all blue, etc.)

let me know!

(right now rockitten red, cardiling yellow and tweesher green, but we can change those without problems)
colors available: red, yellow, green, violet and blue (blue is the default one).

JaskRendix committed:
Merge branch 'Tuxemon:development' into choice

Qiangong2 commented:
That looks great! Thanks for implementing it!

As for the tuxeball colors, I think Rockitten being gray, Cardiling being red, and Tweesher being yellow is the best combination. Gray, red, and yellow fit the Tuxemon better in my opinion.

JaskRendix committed:
choice

JaskRendix commented:
That's great, fixed color order.
Fixed also the environment (background battle), before it was giving **grass** background during the fight vs Professor, now it gives the **interior** background.
You can set different backgrounds (options here: https://wiki.tuxemon.org/Battle_Terrain)
